templates/_base/master/files/container-registries.yaml

```diff
@@ -0,0 +1,30 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/containers/registries.conf"
+contents:
+  inline: |
+    # This is a system-wide configuration file used to
+    # keep track of registries for various container backends.
+    # It adheres to TOML format and does not support recursive
+    # lists of registries.
+
+    # The default location for this configuration file is /etc/containers/registries.conf.
+
+    # The only valid categories are: 'registries.search', 'registries.insecure',
+    # and 'registries.block'.
+
+    [registries.search]
+    registries = ['registry.access.redhat.com', 'docker.io']
+
+    # If you need to access insecure registries, add the registry's fully-qualified name.
+    # An insecure registry is one that does not have a valid SSL certificate or only does HTTP.
+    [registries.insecure]
```

Comment:
As mentioned in https://github.com/openshift/installer/pull/234#discussion_r216755364, can you remove these empty sections?

Reply:
updated